integration/spark/src/main/common/java/io/openlineage/spark/api/QueryPlanVisitor.java

```diff
@@ -30,9 +30,9 @@
  * org.apache.spark.sql.SparkSession} (if set), the {@link org.apache.spark.SparkContext}, and other
  * existing {@link QueryPlanVisitor}s for delegating visitors.
  *
- * <p>Subclasses should use the {@link OpenLineage} client returned from {@link
- * OpenLineageContext#getOpenLineage()} to construct all model objects to that the correct producer
- * and schema URIs are set. For convenience, {@link #outputDataset()} and {@link #inputDataset()}
+ * <p>Subclasses should use the {@link OpenLineage} client returned from {@link OpenLineageContext}
```

Comment:
It might still be helpful to document the method:

```
p>Subclasses should use the {@link OpenLineage} client returned by calling the {@code getOpenLineage()} defined in the class {@link OpenLineageContext}...
```

Reply:
+1 from me

Reply:
Fixed.